src/app/shared/components/template/components/radio-button-grid/radio-button-grid.component.scss

```diff
@@ -77,3 +77,69 @@ $text-color: var(--radio-button-font-color, var(--ion-color-primary));
     }
   }
 }
+
+.radio-buttons[data-variant="flex"] {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-evenly;
+  align-items: center;
+
+  width: 100%;
+  .item {
+    background-color: var(--ion-color-secondary-200);
+    box-shadow: none;
+    border: none;
+
+    width: 140px;
+    height: 140px;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+
+    overflow: hidden;
+    border-radius: 8px;
+    padding: 0px;
+    padding: 12px 14px;
+    margin: 4px 0;
+
+    max-width: calc(7.5rem * 0.9);
+    min-width: calc(6rem * 0.9);
```

Comment:
Where are these values coming from? And do we need to set a `max-` and `min-` width if we've already set the width explicitly above?

Reply:
These are responsiveness values carried over from the old radio-group styling file. The width is explicit but the item should still stretch or shrink according to the length of text.